tsdb/docs/format/wal.md

```diff
@@ -50,6 +50,27 @@ Series records encode the labels that identifies a series and its unique ID.
 └────────────────────────────────────────────┘
 ```
 
+### Metadata records
+
+Metadata records encode the metadata associated with a series.
+
+```
+┌────────────────────────────────────────────┐
+│ type = 1 <1b>                              │
+├────────────────────────────────────────────┤
+│ ┌────────────────────────────────────────┐ │
+│ │ id <uvarint>                           │ │
+│ ├─────────────────────┬──────────────────┤ │
+│ │ len(type) <uvarint> │ type <bytes>     │ │
```

Comment:
type should be an int

Reply:
Sounds good, I'll update `storage.Metadata` to use a uint8 as well.